library/src/lib/calendar/calendar-views/calendar-day-view/calendar-day-view.component.ts

```diff
@@ -537,7 +529,7 @@ export class CalendarDayViewComponent implements OnInit, AfterViewChecked, OnCha
      * Method that returns first letter of every weekday, basing on CalendarI18nDefault. Can be changed by user by
      * providing other class which implements CalendarI18n
      */
-    private getShortWeekDays(): string[] {
+    get shortWeekDays(): string[] {
```

Comment:
I'm not sure about this approach. Every time angular detects any change(scroll, mouse hover, clicks) these array methods will be called. It can be bad for performance. The same thing is used on month view and it wasn't changed.